src/patch_parser.py

```diff
@@ -292,7 +326,7 @@ def parse_comments(email_thread: Message) -> Patchset:
             assert length == len(patches)
         text = 'From: {from_}\nSubject: {subject}\n\n{content}'.format(
             from_=patch.from_, subject=patch.subject, content=patch.content)
-        patch_list.append(Patch(message_id = patch.id,
+        patch_list.append(Patch(message_id=patch.id,
```

Comment:
You have **a lot** of formatting fixes here. If it was just a handful, that would be OK, but with as many as there is, it is distracting from the core purpose of this change. Can you submit the formatting fixes as a separate change?